plugin/src/main/java/org/jetbrains/research/refactorinsight/actions/HideNonFunctionalChangesAction.java

```diff
@@ -0,0 +1,35 @@
+package org.jetbrains.research.refactorinsight.actions;
+
+import com.intellij.diff.FrameDiffTool;
+import com.intellij.diff.tools.util.DiffDataKeys;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.CommonDataKeys;
+import com.intellij.openapi.actionSystem.ex.CheckboxAction;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.editor.FoldRegion;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.research.refactorinsight.folding.ImportFolder;
+import org.jetbrains.research.refactorinsight.folding.RefactoringFolder;
+
+public class HideNonFunctionalChangesAction extends CheckboxAction {
```

Comment:
IDE highlights this class with yellow color and suggests to override 'getActionUpdateThread' and choose 'EDT' or 'BGT'